packages/vaex-core/vaex/expression.py

```diff
@@ -538,12 +538,40 @@ def count(self, binby=[], limits=None, shape=default_shape, selection=False, del
         kwargs['expression'] = self.expression
         return self.ds.count(**kwargs)
 
-    def sum(self, binby=[], limits=None, shape=default_shape, selection=False, delay=False, progress=None):
+    def sum(self, axis=None, binby=[], limits=None, shape=default_shape, selection=False, delay=False, fill_missing=None, fill_missing_item=0, fill_empty=0, progress=None):
         '''Shortcut for ds.sum(expression, ...), see `Dataset.sum`'''
```

Comment:
This docstring should be updated. 

Probably the `df.sum()` should also be updated to include an explanation of the new kwargs.